persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -942,7 +943,8 @@ getColumn getter tableName' [PersistText columnName, PersistText isNullable, Per
                           , cDefault = fmap stripSuffixes d''
                           , cDefaultConstraintName = Nothing
                           , cMaxLen = Nothing
-                          , cReference = ref
+                          -- TODO: Fix cascade reference is ignored
```

Comment:
looks like this is available [here](https://www.postgresql.org/docs/9.0/infoschema-referential-constraints.html)